### ISSUE TITLE
extra - how hard is ipad support?

### ISSUE THREAD

**Issue by aarondunn** [musopen/shiny-dubstep#3]
extra - how hard is ipad support?

its such a nice app, id love to use it on my ipad now :)

**Comment by n3lab** [comment]
Changed app to universal (iphone and ipad). Test it please.

**Comment by aarondunn** [comment]
so you need my udid to test on ipad?

**Comment by aarondunn** [comment]
works anyways, amazing.

**Comment by aarondunn** [comment]
question, are we able to tweak padding so it looks a certain way on iphone vs ipad? i could ask my designer to tweak it so its not so left aligned on both
also he may want to center on ipad more like in Apple Music

**Comment by n3lab** [comment]
yep, we can. ask please designer to provide screens for ipad version.

**Comment by aarondunn** [comment]
FYI @n3lab I got psd's for ipad: https://dl.dropboxusercontent.com/u/452595/iPad-App.zip

It doesn't have to be 100% the same, our current one is pretty close. But feel free to take things from it if helpful.

Should we chat? What remains to launch the app?

**Comment by n3lab** [comment]
Answered via Slack.
